users/models.py

```diff
@@ -19,6 +21,21 @@ class Types(models.IntegerChoices):
     homepage_type = models.SmallIntegerField(choices=Types.choices, default=SWIPE_HOMEPAGE)
 
 
+    def __str__(self):
+        return self.user_id.username
+
+
+    def __repr__(self):
+        return __str__()
+
+
+@receiver(post_save, sender=AuthUserModel)
```

Comment:
Move this into `signals.py` and import the file in `apps.py`'s `ready` function. We'll discuss this this week, anyway.